assets/js/base/components/checkbox-list/index.js

```diff
@@ -99,9 +103,9 @@ const CheckboxList = ( {
 				{ options.map( ( option, index ) => (
 					<Fragment key={ option.value }>
 						<li
-							{ ...shouldTruncateOptions &&
+							{ ...( shouldTruncateOptions &&
```

Comment:
good catch!

Reply:
Actually the post-commit linter caught this and I didn’t notice it until you commented, good job robot 🤖